### PR TITLE
Add the logo to header.js

### PR DESCRIPTION
**Pull request by vormwald**
### Problem

The existing syracuse.io has a logo on its main header bar.  The new version should have one too.

### Solution

Lets add this by adding it header.js. The end result looks like this:

screen shot 2019-02-14 at 10 54 40 am

I also had to tweak some styling to get the header bar to look like the existing site.

### CODE REVIEW

File: gatsby-config.js [hunk 1]
```diff
@@ -21,6 +21,7 @@ module.exports = {
         link: '/resources',
       },
     ],
+    logo: 'logo.png'
```
Comment (vormwald) [inline gatsby-config.js]: This is not used, and unnecessary. 🙅‍♂️

File: src/components/header.js [hunk 2]
```diff
@@ -2,12 +2,14 @@ import React from 'react'
 import styled from 'styled-components'
 
 import { Link } from 'gatsby'
+import Img from 'gatsby-image'
 import PropTypes from 'prop-types'
+import logo from '../images/logo.png';
```
Comment (vormwald) [inline src/components/header.js]: Lets use a SVG instead.

File: src/components/header.js [hunk 3]
```diff
@@ -39,6 +41,13 @@ const MenuItem = styled.li`
   line-height: 20px;
   height: 50px;
 `
+const Logo = styled.div`
+  width: 100px;
+  opacity: 1;
+  transition: opacity 0.5s ease-in-out;
```
Comment (vormwald) [inline src/components/header.js]: necessary?

File: src/components/header.js [hunk 4]
```diff
@@ -53,10 +62,23 @@ const Header = ({ siteTitle, menuLinks }) => (
           </MenuItem>
         ))}
       </NavList>
+      <div style={{width: '100px',
```
Comment (vormwald) [inline src/components/header.js]: Put this in a styled component

Comment (vormwald) [reply src/components/header.js]: call it a `LogoWrapper` or `LogoContainer`

**Comment by lannonbr** [comment]
LGTM!

**Pull request merged**